apps/web/src/state/settings/internal.ts

```diff
@@ -0,0 +1,11 @@
+import { proxy } from 'valtio';
+
+export interface SettingState {
+	_id: string;
+	hideBalance: boolean;
+}
+
+export const settingState = proxy<SettingState>({
+	_id: '',
+	hideBalance: true,
+});
```

Comment:
- if we have new state, that would be use in Mobile (this state will be use in Mobile) - then it should be in `packages/app` instead
- since we put the setting in existed db object, "settingState" is actually the `appState` we already had, I think we better put this `hideBalance` just in the `appState`
- then, remove this `settingState` I guess

Reply:
do you think there will be more settings share among platforms? should we create new `settingsState` instead of putting more settings to `appState`?

Reply:
in that case, it would be an object inside `setting` document (instead of a field like now)

with it, we'll have an object e.g `setting.preferences` (now we have setting.hideBalance), and from within `appState`, we'll have `appState.perferences`.